src/test/java/address/guitests/GuiTestBase.java

```diff
@@ -106,6 +107,12 @@ protected void pressKeyCombo(KeyBinding kb){
         return keys;
     }
 
+    public KeyCodeCombination shortcut(KeyCode keyCode) {
+        if (OsDetector.isOnMac()) {
+            return new KeyCodeCombination(keyCode, KeyCodeCombination.META_DOWN);
+        }
+        return new KeyCodeCombination(keyCode, KeyCodeCombination.CONTROL_DOWN);
+    }
```

Comment:
But it seems unnecessarily long during the tests as we have to create new `KeyCodeCombination`s in the tests, when instead we can just call `shortcut(KeyCode)`.